vestaboard/__init__.py

```diff
@@ -61,6 +62,26 @@ def raw(self, charList):
       for j, char in enumerate(row):
         if not isinstance(char, int):
           raise ValueError(f'Nested lists must contain numbers only - check row {i} char {j} (0 indexed)')
+    if len(charList) > 6:
+      del charList[6:]
```

Comment:
Normally I would advise against modifying the input array (side effects and all), but in this case I think it's a brilliant way around the fact that the `.raw()` function doesn't return anything - clever! 🧠